run_tests.bat

```diff
@@ -0,0 +1,29 @@
+@echo off
+rem Runs the tests locally.
+
+setlocal
+
+set runner_root=%USERPROFILE%\testrunner
+set nunit="%runner_root%\NUnit.ConsoleRunner.3.2.1\tools\nunit3-console.exe"
+
+if not exist %nunit% (
+  rem Grabs the nunit test runner.
+  echo Installing nunit runner.
+  nuget install NUnit.ConsoleRunner -Version 3.2.1 -OutputDirectory "%runner_root%"
+) else (
+  echo Nunit runner already installed.
+)
+
+rem Build the code with the travis configuration.
+pushd Src\Support
+msbuild /p:Configuration=ReleaseTravis GoogleApisClient.sln
```

Comment:
ReleaseTravis misses out quite a few libraries, but none of the net45 libraries, so it's fine.